PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,20 +11,21 @@
 
 ## Description of change
 
-*Please replace with a description about why this change is needed, along with a description of what changed?*
+<!--- *Please replace with a description about why this change is needed, along with a description of what changed?* -->
 
 ## QA steps
 
-*Please replace with how we can verify that the change works?*
+<!--- *Please replace with how we can verify that the change works?* -->
 
 ```sh
 QA steps here
 ```
 
 ## Documentation changes
 
-*Please replace with any notes about how it affects current user workflow? CLI? API?*
+<!--- *Please replace with how we can verify that the change works?* 
+ *Please replace with any notes about how it affects current user workflow? CLI? API?* -->
 
 ## Bug reference
 
-*Please add a link to any bugs that this change is related to.*
+<!--*Please add a link to any bugs that this change is related to.* -->
```

Comment:
Missed a dash here. 

I would remove the the other markdown parts from the comments too, "*" and "##".